apps/examples/websocket/websocket_main.c

```diff
@@ -795,10 +795,6 @@ int websocket_main(int argc, char *argv[])
 		goto error_with_function_failure;
 	}
```

Comment:
Kindly release the allocated memory for "input" during success case as well before returning.

Reply:
That is exactly what we should not be doing, hence the commit. "input"is passed as a parameter to both websocket_client and websocket_server, and both these functions free input (referred to as 'opt' inside). So calling free to input after success should be avoided.

Reply:
@dr-venkman If both websocket_client and websocket_server are racing to free 'input', what happens?